### ISSUE TITLE
NullPointerException with HeaderToProject.addArrayControls

### ISSUE THREAD

**Issue by kayler-renslow** [kayler-renslow/arma-dialog-creator#31]
### Version and Build
1.0.5

### Summary
Nullpointer exception when trying to convert dialog.

### Detail
```
java.lang.NullPointerException
    at com.kaylerrenslow.armaDialogCreator.data.HeaderToProject.addArrayControls(HeaderToProject.java:275)
    at com.kaylerrenslow.armaDialogCreator.data.HeaderToProject.saveToWorkspace(HeaderToProject.java:239)
    at com.kaylerrenslow.armaDialogCreator.data.HeaderToProject.run(HeaderToProject.java:158)
    at com.kaylerrenslow.armaDialogCreator.data.HeaderToProject.convertAndSaveToWorkspace(HeaderToProject.java:65)
    at com.kaylerrenslow.armaDialogCreator.gui.main.popup.projectInit.ImportDialogsDialog$ConvertTask.call(ImportDialogsDialog.java:137)
    at com.kaylerrenslow.armaDialogCreator.gui.main.popup.projectInit.ImportDialogsDialog$ConvertTask.call(ImportDialogsDialog.java:125)
    at javafx.concurrent.Task$TaskCallable.call(Task.java:1423)
    at java.util.concurrent.FutureTask.run(Unknown Source)
    at java.lang.Thread.run(Unknown Source)

THREAD:Arma Dialog Creator JavaFX Thread
```

Header code:
```c++
class cell_phone {
    idd = 3000;
    name= "cell_phone";
    movingEnable = false;
    enableSimulation = 1;
    onLoad = "";

   class controls {

class wrp_cell_phone: Life_RscPicture
{
    idc = 1200;
    text = "textures\cell_phone.paa";
    x = 0.699967 * safezoneW + safezoneX;
    y = 0.398004 * safezoneH + safezoneY;
    w = 0.327946 * safezoneW;
    h = 0.54398 * safezoneH;
};
class wrp_messages: Life_RscButtonMenu
{
    idc = 2400;
    x = 0.811949 * safezoneW + safezoneX;
    y = 0.78899 * safezoneH + safezoneY;
    w = 0.023996 * safezoneW;
    h = 0.0509982 * safezoneH;
    animTextureDefault = "";
    animTextureNormal = "";
    animTextureDisabled = "";
    animTextureOver = "";
    animTextureFocused = "";
    animTexturePressed = "";
    action = "hint ""bouton messages"";";

};
class wrp_keys : Life_RscButtonMenu
{
    idc = 2401;
    text = "";
    x = 0.811948 * safezoneW + safezoneX;
    y = 0.686993 * safezoneH + safezoneY;
    w = 0.023996 * safezoneW;
    h = 0.0509982 * safezoneH;
    animTextureDefault = "";
    animTextureNormal = "";
    animTextureDisabled = "";
    animTextureOver = "";
    animTextureFocused = "";
    animTexturePressed = "";
    action = "hint ""bouton keys"";";
};
class wrp_calendar : Life_RscButtonMenu
{
    idc = 2402;
    text = "";
    x = 0.851942 * safezoneW + safezoneX;
    y = 0.686993 * safezoneH + safezoneY;
    w = 0.023996 * safezoneW;
    h = 0.0509982 * safezoneH;
    animTextureDefault = "";
    animTextureNormal = "";
    animTextureDisabled = "";
    animTextureOver = "";
    animTextureFocused = "";
    animTexturePressed = "";
    action = "hint ""bouton calendar"";";
};
class wrp_autosave : Life_RscButtonMenu
{
    idc = 2403;
    text = "";
    x = 0.851942 * safezoneW + safezoneX;
    y = 0.78899 * safezoneH + safezoneY;
    w = 0.023996 * safezoneW;
    h = 0.0509982 * safezoneH;
    animTextureDefault = "";
    animTextureNormal = "";
    animTextureDisabled = "";
    animTextureOver = "";
    animTextureFocused = "";
    animTexturePressed = "";
    action = "hint ""bouton autosave"";";
};
class wrp_backpack : Life_RscButtonMenu
{
    idc = 2404;
    text = "";
    x = 0.891935 * safezoneW + safezoneX;
    y = 0.686993 * safezoneH + safezoneY;
    w = 0.023996 * safezoneW;
    h = 0.0509982 * safezoneH;
    animTextureDefault = "";
    animTextureNormal = "";
    animTextureDisabled = "";
    animTextureOver = "";
    animTextureFocused = "";
    animTexturePressed = "";
    action = "hint ""bouton backpack"";";
};
class wrp_params : Life_RscButtonMenu
{
    idc = 2405;
    text = "";
    x = 0.891935 * safezoneW + safezoneX;
    y = 0.78899 * safezoneH + safezoneY;
    w = 0.023996 * safezoneW;
    h = 0.0509982 * safezoneH;
    animTextureDefault = "";
    animTextureNormal = "";
    animTextureDisabled = "";
    animTextureOver = "";
    animTextureFocused = "";
    animTexturePressed = "";
    action = "hint ""bouton params"";";
};
class wrp_next_button: Life_RscButtonMenu
{
    idc = 2406;
    x = 0.88858 * safezoneW + safezoneX;
    y = 0.850998 * safezoneH + safezoneY;
    w = 0.0208333 * safezoneW;
    h = 0.04 * safezoneH;
    animTextureDefault = "";
    animTextureNormal = "";
    animTextureDisabled = "";
    animTextureOver = "";
    animTextureFocused = "";
    animTexturePressed = "";
    action = "createDialog ""cell_phone_page2"";";
};
};
};

```

**Comment by kayler-renslow** [comment]
I believe this was fixed a while ago

**Comment by kayler-renslow** [comment]
fixed(?) in release 1.0.6
